packages/contracts/package.json

```diff
@@ -28,6 +28,7 @@
     "echidna": "echidna-test . --config echidna.config.yml",
     "prettier": "forge fmt",
     "deploy": "ts-node ./scripts/deploy/deploy.ts",
+    "deploy:diamond": "ts-node ./scripts/deploy/manager/DiamondInAll.ts",
```

Comment:
Intuitively I feel that this doesn't belong here but I don't have any better ideas.